src/components/Recipe/Recipe.js

```diff
@@ -14,7 +14,7 @@ export default function Recipe(props) {
         .map((recipe) => (
           <li className="recipes_list">
           <Link to={`/${category}/${recipe.fields.slug}`}>
-            <h4>{recipe.fields.name}</h4>
+            <h2 className="hhh">{recipe.fields.name}</h2>
```

Comment:
`hhh` is a very creative name for a class :P 

Please gives something more meaningful, like title, subtitle, h2 or similar.